src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java

```diff
@@ -1699,7 +1699,8 @@ protected void addTranslations()
 		addLand(MonstersLandType.MONSTERS, "Monsters");
 		addLand(TowersLandType.TOWERS, "Towers");
 		add(LandTypePair.FORMAT, "Land of %s and %s");
-		add(SburbHandler.LAND_ENTRY, "Welcome to %s.");
+		add(SburbHandler.CHAT_LAND_ENTRY, "Welcome to %s.");
+		add(SburbHandler.TITLE_LAND_ENTRY, "%s");
```

Comment:
What is the point to this specific lang entry?

Reply:
I was under the impression we wanted to keep it translatable, in which case two arguments are needed unless I can just "%s" into SburbHandler

Reply:
Keeping it translatable? Two arguments are needed? I'm not sure what problem you're trying to solve.

Reply:
Issue resolved, thanks for help!